src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -363,15 +490,16 @@ public string ExportPkcs8PrivateKeyPem()
         ///   <para>The private key is not exportable.</para>
         ///   <para>-or-</para>
         ///   <para>An error occurred while exporting the key.</para>
+        ///   <para>-or-</para>
+        ///   <para><paramref name="pbeParameters"/> does not represent a valid password-based encryption algorithm.</para>
         /// </exception>
-        public byte[] ExportEncryptedPkcs8PrivateKey(ReadOnlySpan<char> password, PbeParameters pbeParameters)
+        public byte[] ExportEncryptedPkcs8PrivateKey(ReadOnlySpan<byte> passwordBytes, PbeParameters pbeParameters)
```

Comment:
Why have you flipped which of the char and byte functions is first?  It's just creating a bunch of noise in this change.

Reply:
I wanted to co-locate the `string` and `ReadOnlySpan<char>` overloads while keeping the `string` overload last since it's just a convenience overload. Not a big deal, so I reverted.